function.k

```diff
@@ -277,13 +287,17 @@ rule [ReturnValue]:
      <result> _ => V </result>
      <callState>
         ...
+        //<abnormalMode> _:Bool => true </abnormalMode>
+        //<abnormalCode> _ => #return# </abnormalCode>
```

Comment:
Why have these been added but commented? If you think you'll need them for a future change, wait until that future change to add them, otherwise the semantics grows larger and larger with things that we think we may need.